Jenkinsfile_CNP

```diff
@@ -136,7 +136,7 @@ withPipeline(type, product, component) {
 
         if (isHearingsPr) {
             env.CASE_TYPE = "Benefit-${CHANGE_ID}"
-            setAatEnvVars();
+            setAatEnvVarsForHearings();
```

Comment:
rather than making a whole new method with a lot of duplicate code I would call `setAatEnvVars()` and then put: `env.TEST_E2E_API_URI = "https://sscs-tribunals-api-pr-${CHANGE_ID}.preview.platform.hmcts.net"` at the end

Reply:
yes we can just override :)